src/server/scripts/World/npcs_special.cpp

```diff
@@ -2192,7 +2192,7 @@ class npc_firework : public CreatureScript
                 // Check if we are near Elune'ara lake south, if so try to summon Omen or a minion
                 if (me->GetZoneId() == ZONE_MOONGLADE)
                 {
-                    if (!me->FindNearestCreature(NPC_OMEN, 100.0f, false) && me->GetDistance2d(omenSummonPos.GetPositionX(), omenSummonPos.GetPositionY()) <= 100.0f)
+                    if (!me->FindNearestCreature(NPC_OMEN, 100.0f, true) && me->GetDistance2d(omenSummonPos.GetPositionX(), omenSummonPos.GetPositionY()) <= 100.0f)
```

Comment:
You can omit that parameter entirely since it's true by default 

```cpp
!me->FindNearestCreature(NPC_OMEN, 100.0f);
```

But I thought that param worked differently - it would ALSO check for dead creatures in addition to alive ones, and not search for only dead creatures